src/aarch64.rs

```diff
@@ -0,0 +1,210 @@
+use libseccomp::*;
```

Comment:
I don't think having two copies of the source code is the way to implement multi-arch support, and I don't think I can accept this PR as-is.

I think there's probably a way to make Rule generic over syscall types, but this is hampered by the fact that Sysno is an enum in an external crate.

Probably doing something like

```
pub struct X86_64Sysno {
  pub sysno: syscalls::x86_64::Sysno
}
pub struct AArchSysno { ... }

trait ESSysno { /* maybe a get_sysno() here or something */}

impl ESSysno for X86_64Sysno {}
impl ESSysno for AArch64Sysno {}

pub struct Rule<S: ESSysno>
```

would work. I'm not sure whether sealing the trait would be the right thing to do or not, would have to think about it.

Reply:
> I don't think having two copies of the source code is the way to implement multi-arch support, and I don't think I can accept this PR as-is.

If can be done much easier, this was Just the easiest way to get compile-time checking for both archs (otherwise, you can only test one, your host arch). I can create another PR then to show that.